src/note/note.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Controller, Get, Post, Put, Delete, Body, Param, Logger } from '@nestjs/common';
+import { NoteService } from './note.service';
+import { Note } from '../interfaces/note'
+
+@Controller('note')
+export class NoteController {
+    constructor(
+        private readonly noteService: NoteService
+    ) { }
+    @Get()
+    all(): Promise<Note[]> {
+        return this.noteService.index()
+    }
+    @Get(':id')
+    one( @Param() params): Promise<Note[]> {
+        return this.noteService.findOne(params.id);
+    }
+    @Post()
+    create( @Body() note: Note): void {
+        this.noteService.create(note);
```

Comment:
tutaj jest niezbędny return, bo bez niego jeśli coś pójdzie nie tak w noteService to użytkownik nigdy się nie dowie :)

Reply:
Poprawione